frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/validations/vm-settings-tab-validation.ts

```diff
@@ -108,6 +110,27 @@ const asVMSettingsFieldValidator = (
     subject: getFieldTitle(iGetFieldKey(field)),
   });
 
+const validateSource: Validator = (field, options): ValidationObject => {
+  const value = iGetFieldValue(field);
+
+  if (value === ProvisionSource.PXE.getValue()) {
+    const { getState, id } = options;
+    const state = getState();
+    const isCreateTemplate = iGetCommonData(state, id, VMWizardProps.isCreateTemplate);
+    if (isCreateTemplate) {
```

Comment:
why is this validation not required for templates?

Reply:
validating templates too